orienteer-core/src/main/java/org/orienteer/core/web/OrienteerBasePage.java

```diff
@@ -79,6 +79,7 @@ public boolean isEnabled(Component component) {
 
 			@Override
 			protected void populateItem(final ListItem<ODocument> item) {
+				item.setRenderBodyOnly(true);
```

Comment:
Commonly it's not recommended because of AJAX. Why it's needed?

Reply:
In bootstrap4 dropdowns use links directly, without external tags.